graphene_sqlalchemy/enums.py

```diff
@@ -0,0 +1,204 @@
+    If the SQLAlchemy Enum is based on a Python Enum, use the name
+    of the Python Enum.  Otherwise, if the SQLAlchemy Enum is named,
+    use the SQL name after conversion to a type name. Otherwise, use
+    the given fallback_name or raise an error if it is empty.
+
+    The Enum value names are converted to upper case if necessary.
+    """
+    if not isinstance(sa_enum, SQLAlchemyEnumType):
+        raise TypeError(
+            "Expected sqlalchemy.types.Enum, but got: {!r}".format(sa_enum)
+        )
+    enum_class = sa_enum.enum_class
+    if enum_class:
+        if all(to_enum_value_name(key) == key for key in enum_class.__members__):
+            return Enum.from_enum(enum_class)
+        name = enum_class.__name__
+        members = [
+            (to_enum_value_name(key), value.value)
+            for key, value in enum_class.__members__.items()
+        ]
+    else:
+        sql_enum_name = sa_enum.name
+        if sql_enum_name:
+            name = to_type_name(sql_enum_name)
+        elif fallback_name:
+            name = fallback_name
+        else:
+            raise TypeError("No type name specified for {!r}".format(sa_enum))
+        members = [(to_enum_value_name(key), key) for key in sa_enum.enums]
+    return Enum(name, members)
+
+
+def enum_for_sa_enum(sa_enum, registry):
+    """Return the Graphene Enum type for the specified SQLAlchemy Enum type."""
+    if not isinstance(sa_enum, SQLAlchemyEnumType):
+        raise TypeError(
+            "Expected sqlalchemy.types.Enum, but got: {!r}".format(sa_enum)
+        )
+    enum = registry.get_graphene_enum_for_sa_enum(sa_enum)
+    if not enum:
+        enum = convert_sa_to_graphene_enum(sa_enum)
+        registry.register_enum(sa_enum, enum)
+    return enum
+
+
+def enum_for_field(obj_type, field_name):
+    """Return the Graphene Enum type for the specified Graphene field."""
+    from .types import SQLAlchemyObjectType
+
+    if not isinstance(obj_type, type) or not issubclass(obj_type, SQLAlchemyObjectType):
+        raise TypeError(
+            "Expected SQLAlchemyObjectType, but got: {!r}".format(obj_type))
+    if not field_name or not isinstance(field_name, str):
+        raise TypeError(
+            "Expected a field name, but got: {!r}".format(field_name))
+    registry = obj_type._meta.registry or get_global_registry()
+    orm_field = registry.get_orm_field_for_graphene_field(obj_type, field_name)
+    if orm_field is None:
+        raise TypeError("Cannot get {}.{}".format(obj_type._meta.name, field_name))
+    if not isinstance(orm_field, Column):
+        raise TypeError(
+            "{}.{} does not map to model column".format(obj_type._meta.name, field_name)
+        )
+    sa_enum = orm_field.type
+    if not isinstance(sa_enum, SQLAlchemyEnumType):
+        raise TypeError(
+            "{}.{} does not map to enum column".format(obj_type._meta.name, field_name)
+        )
+    enum = registry.get_graphene_enum_for_sa_enum(sa_enum)
+    if not enum:
```

Comment:
Just to clarify, is that ok if I delete this in the PR for https://github.com/graphql-python/graphene-sqlalchemy/issues/209?

Reply:
Sure, if you think #209 makes this unnecessary.